experimental/InvariantTheory/src/InvariantTheory.jl

```diff
@@ -195,12 +212,18 @@ end
 
 function Base.show(io::IO, R::RepresentationLinearlyReductiveGroup)
     io = pretty(io)
-    @assert group(group(R))[1] == :SL
-    println(io, "Representation of ", group(group(R))[1], group(group(R))[2])
-    if R.sym_deg[1]
-        print(io, Indent(), "on symmetric forms of degree ", R.sym_deg[2])
-        print(io, Dedent())
+    if isdefined(group(R), :group) && group(group(R))[1] == :SL
```

Comment:
`group(group(R))` looks really weird. I had to look up what the "out" group does. 

I suggest to replace that `group` method by something like this:
```
group_type(R::RepresentationLinearlyReductiveGroup) = isdefined(R, :group) ? R.group[1] : nothing
group_dim(R::RepresentationLinearlyReductiveGroup) = R.group[2]
```
then you can change the code here to something like
```suggestion
    if group_type(group(R))[1] == :SL
```